egs/wsj/s5/steps/tfrnnlm/rnnlm.py

```diff
@@ -0,0 +1 @@
+lstm.py
```

Comment:
do you think this link is needed and helpful?   why not just call the script we want directly?